eng/Version.Details.xml

```diff
@@ -236,9 +250,19 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>36b148348ee8312f6369c0c56b0d0fe07deec603</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="0.1.0-alpha.1.21318.1">
```

Comment:
Do we need to add a source-build -> SDK subscription for this?

Reply:
Yes, I will add post merging.